test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -5449,8 +5451,8 @@ TEST_F(AtenXlaTensorTest, TestBatchNorm1D) {
 
 TEST_F(AtenXlaTensorTest, TestBatchNorm2D) {
   int num_features = 3;
-  torch::Tensor input = torch::rand({2, num_features, 4, 4},
-                                    torch::TensorOptions(torch::kFloat));
+  torch::Tensor input =
```

Comment:
Did you install and use `clang-format-7`?
That is our new standard.
The one in our workstations is version 9 or a trunk version.

Reply:
Ok, I'll run `clang-format-7` instead.

Reply:
> Ok, I'll run `clang-format-7` instead.

With the project's .clang-format file (-style=file from a path where .clang-format is parented).

Reply:
It formats it the same.